src/gsCore/gsMultiPatch_.cpp

```diff
@@ -15,8 +15,9 @@ namespace gismo
   void pybind11_init_gsMultiPatch(py::module &m)
   {
     using Base = gsFunctionSet<real_t>;
+    using BaseB = gsBoxTopology;
     using Class = gsMultiPatch<real_t>;
-    py::class_<Class,Base>(m, "gsMultiPatch")
+    py::class_<Class,Base, BaseB>(m, "gsMultiPatch")
```

Comment:
using Base1 = gsFunctionSet<real_t>;
using Base2 = gsBoxTopology;

would be clearer than BaseB.